lib/easypost/models/smart_rate.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# The SmartRate object contains details about the SmartRate requests.
+class EasyPost::Models::SmartRate < EasyPost::Models::EasyPostObject
```

Comment:
thought: there is no such thing as an actual "SmartRate" that can be identified via ID. For that reason, I don't think we should include a model. A model is for a first-class EasyPost object that can be identified via either/or the `object` or `id` fields.

Reply:
yea, i realized that after implementing in PHP client lib. Removed